### PR TITLE
[graphite] add pod security policy

### PR DESCRIPTION
**Pull request by davidphay**
#### What this PR does / why we need it:
Add pod security policy support

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

**Comment by davidphay** [comment]
@monotek Hello can you take a look please ?

**Comment by davidphay** [comment]
@rpahli Hello I rebase the branch it should be good now.

File: charts/graphite/templates/_helpers.tpl [hunk 1]
```diff
@@ -30,3 +30,10 @@ Create chart name and version as used by the chart label.
 {{- define "graphite.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Define the service Account name
+*/}}
+{{- define "graphite.serviceAccount.name" -}}
+{{ default "graphite" .Values.serviceAccount.accountName }}
```
Comment (rpahli) [inline charts/graphite/templates/_helpers.tpl:38]: the value `accountName` is not documented. Pls also add this field to the `values.yaml` file and comment it out.

**Review by rpahli — CHANGES_REQUESTED** [review]

**Comment by davidphay** [comment]
@rpahli It should be good now

**Review by rpahli — APPROVED** [review]

**Pull request merged**